.github/ISSUE_TEMPLATE/enhancement.md

```diff
@@ -0,0 +1,21 @@
+---
+name: Enhancement
+about: A request for a new feature, or to enhance existing functionality
+labels: enhancement
+---
+<!--
+
+Please describe the enhancement you would like to see implemented in
+tendermint-rs. Give as much context as possible.
+
+-->
+
+**When can we consider this to be "done"?**
```

Comment:
Same comment as above: clarifying "when".

```suggestion
**Describe how the feature looks like and works in practice**
```

To be honest, this second item might be redundant here. What's the difference between "describing the definition of done" and "describing the enhancement you would like to see implemented". If it's only the level of detail, maybe we can remove this second question and just add "Please be as detailed as you can" to the first one.

Reply:
agree w/ @greg-szabo here

Reply:
Same as the earlier comment - is the comment below the heading not clear enough here?